src/mscorlib/src/System/Text/UTF32Encoding.cs

```diff
@@ -516,6 +534,7 @@ internal override unsafe int GetCharCount(byte* bytes, int count, DecoderNLS bas
 
             // For fallback we may need a fallback buffer
             DecoderFallbackBuffer fallbackBuffer = null;
+            char* charsForFallback;
```

Comment:
Is this used? 

In the log i only see src\System\Text\UTF32Encoding.cs(537,19): warning CS0168: The variable 'charsForFallback' is declared but never used [D:\j\workspace\x64_release_w---0575cb46\src\mscorlib\System.Private.CoreLib.csproj]